Rocket.Chat/Resources/cs.lproj/Localizable.strings

```diff
@@ -114,6 +114,8 @@
 "auth.email_auth" = "e-mail"; // TODO
 "auth.login_service_prefix" = "Continue with "; // TODO
 
+"auth.more" = "More"; // TODO
```

Comment:
Move these strings to VoiceOver.strings

https://github.com/RocketChat/Rocket.Chat.iOS/blob/develop/Rocket.Chat/Resources/en.lproj/VoiceOver.strings

Reply:
Done. 👍